Gruntfile.js

```diff
@@ -1,8 +1,9 @@
 module.exports = function(grunt) {
 
-  var reporter = 'Nyan';
-  if (grunt.option('ci')) {
-    reporter = 'Dot';
+  var reporter = 'Dot';
```

Comment:
change the default report to be 'Dot' because 'Nyan' slows down phantomjs so much that some of the tests that do GET requests to the server timeout.